app/(landing-layout)/_component/FeaturesSection.tsx

```diff
@@ -0,0 +1,131 @@
+    cta: "Learn more",
+    className: "lg:row-start-2 lg:row-end-3 lg:col-start-2 lg:col-end-3",
+  },
+  {
+    Icon: CalendarIcon as React.ElementType,
+    name: "Calendar",
+    description: "Use the calendar to filter your files by date.",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-1 lg:row-end-2 lg:col-start-3 lg:col-end-4",
+  },
+  {
+    Icon: BellIcon as React.ElementType,
+    name: "Notifications",
+    description:
+      "Get notified when someone shares a file or mentions you in a comment.",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-2 lg:row-end-3 lg:col-start-3 lg:col-end-4",
+  },
+  {
+    Icon: IconGlobe as React.ElementType,
+    name: "A customized journey",
+    description: "Find destinations for your next trip",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-3 lg:row-end-4 lg:col-start-1 lg:col-end-3",
+  },
+  {
+    Icon: IconMap as React.ElementType,
+    name: "Plan your itinerary",
+    description: "Create your itinerary from start to finish",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-3 lg:row-end-5 lg:col-start-3 lg:col-end-4",
+  },
+  {
+    Icon: IconCompass as React.ElementType,
+    name: "Activity suggestions",
+    description: "Receive activity suggestions based on your preferences",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-4 lg:row-end-5 lg:col-start-1 lg:col-end-2",
+  },
+  {
+    Icon: IconTimeline as React.ElementType,
+    name: "Time estimate calculation",
+    description: "Calculate the estimated time for each activity",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-4 lg:row-end-5 lg:col-start-2 lg:col-end-3",
+  },
+  {
+    Icon: IconTicket as React.ElementType,
+    name: "Book your tickets",
+    description: "Book your flight, train, or bus tickets",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-5 lg:row-end-6 lg:col-start-1 lg:col-end-3",
+  },
+  {
+    Icon: IconDownload as React.ElementType,
+    name: "Download your itinerary",
+    description: "Download your itinerary for offline use",
+    href: "/",
+    cta: "Learn more",
+    className: "lg:row-start-5 lg:row-end-6 lg:col-start-3 lg:col-end-4",
+  },
+];
+
+export const FeaturesSection = () => {
+  return (
+    <>
+      <Container size="xl" mt="sm">
+        <Stack w="100%" maw={400}>
+          <Title order={1} c="teal" fw={900}>Features</Title>
+        </Stack>
+        <Stack w="100%" maw={600}>
+          <Text size="lg" c="teal" my="sm">
+            Here is the list of features included in our application and which you can discover.
+          </Text>
+        </Stack>
+        <BentoGrid className="lg:grid-rows-3 mb-3">
+          {features.map((feature) => (
+            <BentoCard key={feature.name} {...feature} />
+          ))}
+        </BentoGrid>
+        <Divider size={2} color="teal" mt="xl" />
+      </Container>
+    </>
```

Comment:
Here the artifact are useless

```suggestion
      <Container size="xl" mt="sm">
        <Stack w="100%" maw={400}>
          <Title order={1} c="teal" fw={900}>Features</Title>
        </Stack>
        <Stack w="100%" maw={600}>
          <Text size="lg" c="teal" my="sm">
            Here is the list of features included in our application and which you can discover.
          </Text>
        </Stack>
        <BentoGrid className="lg:grid-rows-3 mb-3">
          {features.map((feature) => (
            <BentoCard key={feature.name} {...feature} />
          ))}
        </BentoGrid>
        <Divider size={2} color="teal" mt="xl" />
      </Container>
```